src/courtroom.cpp

```diff
@@ -5123,7 +5227,7 @@ void Courtroom::set_text_color_dropdown()
     color_row_to_number.append(c);
   }
   for (int c = 0; c < max_colors; ++c) {
-    QColor color = ao_app->get_chat_color("c" + QString::number(c), "default");
+    QColor color = ao_app->get_chat_color("c" + QString::number(c), "");
```

Comment:
???

Reply:
This way, it doesn't head to base/misc/default/chat_config.ini or base/themes/current_theme/default/chat_config.ini, prioritizing the current theme's chat_config.ini and only falling back to the base/misc/default/config.ini monstrosity as a last resort